_health-care/_js/utils/options-for-select.js

```diff
@@ -237,6 +237,18 @@ const maritalStatuses = [
   'Divorced'
 ];
 
+const branchesServed = [
+  { value: 'army', label: 'Army' },
```

Comment:
I think for these you can just structure them like `maritalStatuses` above: 
```javascript
const branchesServed = [
  'Army',
  'Airforce',
  'Marine',
  'Navy'
]
```
I don't think we need to specify different stored values in this case.